src/masternodeman.cpp

```diff
@@ -651,55 +651,56 @@ std::vector<std::pair<int, CMasternode> > CMasternodeMan::GetMasternodeRanks(int
     return vecMasternodeRanks;
 }
 
+const ProcessResult ProcessResult::OK = ProcessResult(true, 0);
+const ProcessResult ProcessResult::FAIL = ProcessResult(true, 0);
```

Comment:
Shouln't make a difference with banscore 0, but `FAIL` is probably `ProcessResult(false, 0)`, no?

Reply:
yeah.